compiler/rustc_attr_parsing/src/attributes/cfg.rs

```diff
@@ -1,6 +1,4 @@
-//! Parsing and validation of builtin attributes
-
-use rustc_ast::{self as ast, LitKind, MetaItem, MetaItemInner, MetaItemKind, MetaItemLit, NodeId};
+use rustc_ast::{LitKind, MetaItem, MetaItemInner, MetaItemKind, MetaItemLit, NodeId};
 use rustc_ast_pretty::pprust;
```

Comment:
cfg is mostly unchanged up to some import changes. I think eve nsome of those might be superfluous